pnpm-lock.yaml

```diff
@@ -0,0 +1,4643 @@
+lockfileVersion: '6.0'
```

Comment:
do we need to check this in?

Reply:
No, we don't have to. I just moved it to .gitignore file.